multiqc/multiqc.py

```diff
@@ -381,14 +382,34 @@ def run(
     # Check that we're running the latest version of MultiQC
     if config.no_version_check is not True:
         try:
-            response = urlopen("http://multiqc.info/version.php?v={}".format(config.short_version), timeout=5)
-            remote_version = response.read().decode("utf-8").strip()
-            if version.StrictVersion(re.sub("[^0-9\.]", "", remote_version)) > version.StrictVersion(
-                re.sub("[^0-9\.]", "", config.short_version)
+            # Fetch the version info from the API
+            meta = {
+                "version_multiqc": config.short_version,
+                "version_python": platform.python_version(),
+                "operating_system": platform.system(),
+                "is_docker": os.path.exists("/.dockerenv"),
+                "is_singularity": os.path.exists("/.singularity.d"),
+                "is_conda": os.path.exists(os.path.join(sys.prefix, "conda-meta")),
+                "ci_environment": os.getenv("CI", False),
```

Comment:
Should probably force this to be a boolean, in case it's ever set as a string. Will help clean data validation if nothing else.